service/history/configs/config.go

```diff
@@ -303,6 +304,8 @@ type Config struct {
 	ArchivalProcessorMaxPollIntervalJitterCoefficient   dynamicconfig.FloatPropertyFn
 	ArchivalProcessorUpdateAckInterval                  dynamicconfig.DurationPropertyFn
 	ArchivalProcessorUpdateAckIntervalJitterCoefficient dynamicconfig.FloatPropertyFn
+	ArchivalProcessorArchiveDelay                       dynamicconfig.DurationPropertyFn
+	ArchivalProcessorArchiveDelayJitterCoefficient      dynamicconfig.FloatPropertyFn
```

Comment:
for simplicity, let's use 1 for add jitter.

Reply:
Got it, removed